packages/library/src/__tests__/transaction-confirmation-strategy-timeout-test.ts

```diff
@@ -0,0 +1,33 @@
+import { getTimeoutPromise } from '../transaction-confirmation-strategy-timeout';
+
+describe('getTimeoutPromise', () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+    });
+    it.each`
+        commitment     | defaultTimeoutMs
+        ${'processed'} | ${30_000}
+        ${'confirmed'} | ${60_000}
+        ${'finalized'} | ${60_000}
+    `('pends for $defaultTimeoutMs when the commitment is `$commitment`', async ({ commitment, defaultTimeoutMs }) => {
```

Comment:
TIL that Jest parses tables!?

Reply:
Whatever you do, don't look at `accounts-test.ts` 😉